src/harness/testcases/WINNF_FT_S_FDB_testcase.py

```diff
@@ -1188,6 +1193,7 @@ def generate_FDB_8_default_config(self, filename):
     config = {
         'registrationRequests': [device_a, device_b],
         'grantRequests': [grant_g_a, grant_g_b],
+        'cpasInitiateTimer': 100,
```

Comment:
We will wait until a specified time has passed, not a specific duration. This time should be specified at the top of this file. You can use 2am Pacific for now but please put a "TODO" comment nearby that it will need to be changed once SAS Admins agree on the timing.

Reply:
Added   sleep until daily activities scheduled